signer/tests/integration/postgres.rs

```diff
@@ -1364,6 +1367,79 @@ async fn get_signers_script_pubkeys_returns_non_empty_vec_old_rows() {
     signer::testing::storage::drop_db(db).await;
 }
 
+/// The [`DbRead::get_last_encrypted_dkg_shares`] function is supposed to
+/// fetch the last encrypted DKG shares stored in the database.
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[tokio::test]
+async fn get_last_encrypted_dkg_shares_gets_most_recent_shares() {
+    let db_num = DATABASE_NUM.fetch_add(1, Ordering::SeqCst);
+    let db = testing::storage::new_test_database(db_num, true).await;
+
+    let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+
+    // We have an empty database, so we don't have any DKG shares there.
+    let no_shares = db.get_last_encrypted_dkg_shares().await.unwrap();
+    assert!(no_shares.is_none());
+
+    // Let's create some random DKG shares and store them in the database.
+    // When we fetch the last one, there is only one to get, so nothing
+    // surprising yet.
+    let shares: model::EncryptedDkgShares = fake::Faker.fake_with_rng(&mut rng);
+    db.write_encrypted_dkg_shares(&shares).await.unwrap();
+
+    let stored_shares = db.get_last_encrypted_dkg_shares().await.unwrap();
+    assert_eq!(stored_shares.as_ref(), Some(&shares));
+
+    // Now let's pretend that we somehow insert into the database some
```

Comment:
This works but wouldn't it have been a little easier to just `db.write_encrypted_dkg_shares()` a few times with a few ms inbetween and tested that the newest row is consistently returned? :see_no_evil:

Reply:
yeah that's better. I'll do that.

Reply:
Done in https://github.com/stacks-network/sbtc/pull/655/commits/0362a3eb90a0ea06d227866197ddec65e6e092d6